testfixtures/logcapture.py

```diff
@@ -225,15 +289,18 @@ def check_present(self, *expected, **kw):
             expected = list(expected)
             matched = []
             unmatched = []
+            i = 0
             for entry in actual:
```

Comment:
I don't think we'll end up with the separate `asserted` list, but the way to do this in Python is:
```
for i, entry in enumerate(actual):
```